app/default/data.py

```diff
@@ -99,21 +99,19 @@ def get_applications_for_account(account_id, as_dict=False):
         return application_response
 
 
-def get_fund_data(fund_id, as_dict=False):
+def get_fund_data(fund_id, language=None, as_dict=False):
     fund_request_url = Config.GET_FUND_DATA_ENDPOINT.format(fund_id=fund_id)
-    language = {"language": get_lang()}
```

Comment:
I think we need to keep this for other language switching scenarios, but we should only do this if no language is passed-in to the function.

Reply:
@gidsg I am passing the  language from application and the application will always have a language so do we need it?

Reply:
I see what you mean. Function updated @gidsg 
Thank you